src/main/java/org/generationcp/middleware/api/germplasm/GermplasmServiceImpl.java

```diff
@@ -320,26 +333,113 @@ private void updateGermplasm(final Germplasm germplasm, final GermplasmUpdateDTO
 				Optional.ofNullable(Integer.parseInt(germplasmUpdateDTO.getCreationDate()));
```

Comment:
Can just be `Optional.of(Integer.parseInt(germplasmUpdateDTO.getCreationDate()))`;